aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsPresigningDecorator.kt

```diff
@@ -166,7 +166,7 @@ class AwsInputPresignedMethod(
         val makeOperationFn = "_make_presigned_operation"
 
         val protocol = section.protocol
-        MakeOperationGenerator(
+        ClientMakeOperationGenerator(
```

Comment:
Since there isn't a `MakeOperationGenerator` in core, shouldn't this just be named `MakeOperationGenerator`?